resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
And actually it might be nice to use the transformer here too so that we only deal with formatting user responses in one place:

```
$item->user = new UserTransformer($user);
```